.travis.yml

```diff
@@ -43,7 +43,8 @@ jobs:
       python: 'pypy3'
 
     - env: TOXENV=py35-xdist
-      python: '3.5'
+      dist: trusty
+      python: '3.5.0'
```

Comment:
Based on my experience with Hypothesis, we need tests to ensure that Pytest's typing-related things don't regress on `3.5.0` *or* `3.5.x`, meaning an additional environment rather than pinning to `.0`.

Reply:
We have 3.5.x in Azure, no?
btw: Django only supports the latest patch releases of Python versions.